bot/seasons/christmas/__init__.py

```diff
@@ -2,15 +2,17 @@
 
 
 class Christmas(SeasonBase):
+    """
+    We are getting into the festive spirit with a new server icon, new
+    bot name and avatar, and some new commands for you to check out!
+
+    No matter who you are, where you are or what beliefs you may follow,
+    we hope every one of you enjoy this festive season!
+    """
     name = "christmas"
+    greeting = "Happy Holidays!"
+    colour = 0x1f8b4c
     start_date = "01/12"
     end_date = "31/12"
-    bot_name = "Santabot"
-
-    def __init__(self, bot):
-        self.bot = bot
-
-    @property
-    def bot_avatar(self):
-        with open(self.avatar_path("christmas.png"), "rb") as avatar:
-            return bytearray(avatar.read())
+    bot_name = "Merrybot"
```

Comment:
Could we have an additional line here, I know properties aren't exactly the same as functions and whatnot but it would feel a lot cleaner imo.
```suggestion

    bot_name = "Merrybot"
```

Reply:
I don't see the logical grouping you're using in this case.